lib/scheduler.pm

```diff
@@ -122,10 +121,15 @@ sub load_yaml_schedule {
     if (my $yamlfile = get_var('YAML_SCHEDULE')) {
         my $schedule      = $ypp->load_file($root_project_dir . $yamlfile);
         my %schedule_vars = parse_vars($schedule);
+        my $test_context_instance = undef;
         while (my ($var, $value) = each %schedule_vars) { set_var($var, $value) }
         my @schedule_modules = parse_schedule($schedule);
         parse_test_suite_data($schedule);
-        loadtest($_) for (@schedule_modules);
+        if ($schedule_vars{TEST_CONTEXT}) {
+            my $cxt = 'OpenQA::Test::' . $schedule_vars{TEST_CONTEXT};
```

Comment:
Also, I'm not sure where you got the requirement that context has to be in `OpenQA::Test` namespace, wicked tests do not follow this and I don't think that it's necessary.

Reply:
My bad. i thought for some reason that the TestContext was also there, and i did not take another look. I will fix it

Reply:
done